airbyte-webapp/src/components/EntityTable/components/StatusCell.tsx

```diff
@@ -1,30 +1,30 @@
 import React from "react";
 
-import { SchemaChange } from "core/request/AirbyteClient";
+import { SchemaChange, WebBackendConnectionListItem } from "core/request/AirbyteClient";
 import { FeatureItem, useFeature } from "hooks/services/Feature";
 
 import { ChangesStatusIcon } from "./ChangesStatusIcon";
 import styles from "./StatusCell.module.scss";
 import { StatusCellControl } from "./StatusCellControl";
 
 interface StatusCellProps {
-  allowSync?: boolean;
   hasBreakingChange?: boolean;
   enabled?: boolean;
   isSyncing?: boolean;
   isManual?: boolean;
   id: string;
   schemaChange?: SchemaChange;
+  connection: WebBackendConnectionListItem;
 }
 
 export const StatusCell: React.FC<StatusCellProps> = ({
   enabled,
   isManual,
   id,
   isSyncing,
-  allowSync,
   schemaChange,
   hasBreakingChange,
+  connection,
```

Comment:
Now that this is receiving/passing along the entire connection object we can probably weed out some of these other props I would think.

Reply:
Also, it looks like these are a `ConnectionListItem` not `WebBackendConnectionRead`.  Maybe naming it `connectionListItem` would be more clear that we don't have an entire connection object here?

Reply:
> Now that this is receiving/passing along the entire connection object we can probably weed out some of these other props I would think.

True, I think we can reduce quite a few of them! I would prefer to address this in a follow-up PR if you don't mind. Filed an issue here: https://github.com/airbytehq/airbyte/issues/22600

> Also, it looks like these are a ConnectionListItem not WebBackendConnectionRead. Maybe naming it connectionListItem would be more clear that we don't have an entire connection object here?

Hm, I didn't think about that. I generally rely on typescript to tell me what the actual underlying type is. Agreed that people probably would assume it's a `WebBackendConnectionRead` by the variable name, but I would say that both `WebBackendConnectionListItem` and `WebBackendConnectionRead` describe a connection.